### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by kinow**
This is a small change with no associated Issue.

Still cannot update `eslint-config-vuetify       ^0.5.0  →   ^0.6.1` due https://github.com/vuetifyjs/eslint-config-vuetify/issues/7

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Already covered by existing tests.
- [x] No change log entry required (why? e.g. invisible to users).
- [x] No documentation update required.

### CODE REVIEW

**Comment by kinow** [comment]
Moving to draft. New linter breaks the build with new Vue rules.

**Comment by kinow** [comment]
Suppressed linter errors in the dashboard.vue component. We are using the template slots as per Vuetify docs: https://vuetifyjs.com/en/components/data-tables/#item

But looks like the linter gets confused because the slots name are "item.column-name1", "item.column-name2", etc.

File: src/components/cylc/workflow/Lumino.vue [hunk 1]
```diff
@@ -151,6 +151,7 @@ export default {
      * }}
      */
     onWidgetActivated (customEvent) {
+      // eslint-disable-next-line vue/custom-event-name-casing
       this.$emit('lumino:activated', customEvent.detail)
```
Comment (oliver-sanders) [inline src/components/cylc/workflow/Lumino.vue:155]: Funny, the docs seem to suggest that this pattern is correct?

https://eslint.vuejs.org/rules/custom-event-name-casing.html#rule-details

**Review by oliver-sanders — APPROVED** [review]

File: src/components/cylc/workflow/Lumino.vue (hunk 1 above)
Comment (kinow) [reply src/components/cylc/workflow/Lumino.vue:155]: Yup! I guess that's a bug in their linter rule. Another rule failed too, where we had a `v-data-table` with a `v-slot="item.name"` I think. It complained about the slot name, as if it didn't recognize `item.name` and `item.age`, etc, were different slot names :man_shrugging:

Comment (oliver-sanders) [reply src/components/cylc/workflow/Lumino.vue:155]: Yes, was just taking a look through the rule examples and it looks like what we have is compliant.

**Review by hjoliver — APPROVED** [review]
LGTM, but would it be worth adding a comment to review the linter disable directives on future Vue updates, in case they fix the rules?

**Comment by kinow** [comment]
> LGTM, but would it be worth adding a comment to review the linter disable directives on future Vue updates, in case they fix the rules?

Good idea. Rebased, then added the `TODO` markers, with `#510` too, so that we can quickly find the issue/PR with context about that note (to save someone from having to go through `git log`)

**Review by hjoliver — APPROVED** [review]

**Pull request merged**